backend/routers/memories.py

```
@@ -1,11 +1,12 @@
-from fastapi import APIRouter, Depends, HTTPException
+from fastapi import APIRouter, Depends, HTTPException, Body
```

Ruff

`fastapi.Body` imported but unused

Remove unused import: `fastapi.Body`

(F401)